usr/src/cmd/zpool/zpool_main.c

```diff
@@ -57,6 +57,10 @@
 
 #include "statcommon.h"
 
+#ifndef	MAX
+#define	MAX(x, y)	((x) > (y) ? (x) : (y))
+#endif	/* MAX */
```

Comment:
No idea why I didn't find it on my first search, turns out it sits in <sys/sysmacros.h> and isn't ifdef'ed to _KERNEL, so I'll pull it in from there.